css/styles.css

```diff
@@ -0,0 +1,125 @@
+.nav-bar {
+  font-family: "Josefin Slab", sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  margin-top: 2rem;
+  color: #4f6f52;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-size: 24px;
```

Comment:
Per [effective-css](https://lms.learningfuze.com/code-guides/Learning-Fuze/curriculum/effective-css), use rem for font-size!